src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,413 @@
+    rounded = 'md',
+    additionalClasses = [],
+    activeAdditionalClasses = [],
+    prevNextAdditionalClasses = [],
+    containerAdditionalClasses = [],
+    prev = 'Previous',
+    next = 'Next',
+    baseUrl = '/',
+    page = 1,
+    numOfPages = 1 }) => {
+    const sizeClass = {
+        sm: 'w-6 h-6 md:w-8 md:h-8 text-sm',
+        md: 'w-8 h-8 md:w-10 md:h-10 text-sm md:text-base',
+        lg: 'w-10 h-10 md:w-12 md:h-12 text-md md:text-lg',
+    };
+    const sizeClass2 = {
+        sm: 'w-16 h-6 md:w-20 md:h-8 text-sm',
+        md: 'w-20 h-8 md:w-24 md:h-10 text-sm md:text-base',
+        lg: 'w-24 h-10 md:w-28 md:h-12 text-md md:text-lg',
+    };
+    const bgClasses = {
+        primary: 'bg-white hover:bg-primary text-black hover:text-white',
+        secondary: 'bg-white hover:bg-secondary text-black hover:text-white',
+        info: 'bg-white hover:bg-info text-black',
+        light: 'bg-white hover:bg-light text-black',
+        dark: 'bg-white hover:bg-dark text-black hover:text-white',
+        transparent: 'bg-transparent text-black',
+    };
+    const activeBgClasses = {
+        primary: 'bg-primary text-white',
+        secondary: 'bg-secondary text-white',
+        info: 'bg-info text-black',
+        light: 'bg-light text-black',
+        dark: 'bg-dark text-white',
+        transparent: 'bg-transparent text-black',
+    };
+    const borderVariantClasses = {
+        primary: 'border border-primary',
+        secondary: 'border border-secondary',
+        info: 'border border-info',
+        light: 'border border-light',
+        dark: 'border border-dark',
+        transparent: 'border border-transparent',
+        black: 'border border-black',
+        white: 'border border-white',
+    };
+    const borderClasses = {
+        primary: ['border', borderVariantClasses[borderVariant], 'hover:border-primary'].join(' '),
+        secondary: ['border', borderVariantClasses[borderVariant], 'hover:border-secondary'].join(' '),
+        info: ['border', borderVariantClasses[borderVariant], 'hover:border-info'].join(' '),
+        light: ['border', borderVariantClasses[borderVariant], 'hover:border-light'].join(' '),
+        dark: ['border', borderVariantClasses[borderVariant], 'hover:border-dark'].join(' '),
+        transparent: ['border', borderVariantClasses[borderVariant], 'hover:border-transparent'].join(' '),
+    };
+    const activeBorderClasses = {
+        primary: 'border border-primary',
+        secondary: 'border border-secondary',
+        info: 'border border-info',
+        light: 'border border-light',
+        dark: 'border border-dark',
+        transparent: 'border border-transparent',
+    };
+    const roundedClass = {
+        none: 'rounded-none',
+        sm: 'rounded-sm',
+        md: 'rounded-md',
+        lg: 'rounded-lg',
+        full: 'rounded-full',
+    };
+    const safePage = Math.min(Math.max(page, 1), numOfPages);
+
+    return (
+        <div className={[
+            'bg-white',
```

Comment:
Can we remove the default `bg-white` class?
It causes a situation like below, when defaults used
![image](https://user-images.githubusercontent.com/645716/150322320-0470697c-b8fd-43c1-b8da-e2d60d569baa.png)

Reply:
yes, we can